packages/server/src/auth/auth.module.ts

```diff
@@ -5,18 +5,33 @@ import { AuthResolver } from './auth.resolver';
 import { PrismaService } from '../prisma/prisma.service';
 import { UserModule } from '../user/user.module';
 import { PassportModule } from '@nestjs/passport';
-import { JwtModule } from '@nestjs/jwt';
+import { JwtModule, JwtModuleOptions } from '@nestjs/jwt';
 import { JwtStrategy } from './jwt.strategy';
+import { ConfigModule } from '@nestjs/config';
+import { ConfigService } from '@nestjs/config';
 import { ProjectModule } from '../project/project.module';
 
 @Module({
   imports: [
     UserModule,
     PassportModule,
+    ConfigModule,
```

Comment:
Could use this code on App Module, then don't need to import it on sub modules
```suggestion
    ConfigModule.forRoot({ isGlobal: true })
```